recipe/meta.yaml

```diff
@@ -1,26 +1,27 @@
-{% set version = "3.3" %}
+{% set version = "3.4" %}
 
 package:
   name: idna
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/i/idna/idna-{{ version }}.tar.gz
-  sha256: 9d643ff0a55b762d5cdb124b8eaa99c66322e2157b69160bc32796e824360e6d
+  sha256: 814f528e8dead7d329833b91c5faa87d60bf71824cd12a7530b5526063d02cb4
 
 build:
-  noarch: python
   number: 0
   script: {{ PYTHON }} -m pip install . -vvv
+  skip: True  # [py<36]
 
 requirements:
   host:
+    - flit-core
     - python
     - pip
     - wheel
     - setuptools
```

Comment:
`setuptools` is no more needed as a build backend, see https://github.com/kjd/idna/blob/44fa5a0a7f7e3a5051b4d3e2ae7957c0494cd708/pyproject.toml#L3
But the maintainers still use it for legacy users. So we can add a comment for `setuptools`:
```
  # setuptools is needed for legacy users, see https://github.com/kjd/idna/commit/9234d29b38e56e863db03af5980819c031221725
  - setuptools
```

Reply:
Added comment